src/lib/Polykey.ts

```diff
@@ -17,21 +18,22 @@ class Polykey {
     polykeyPath: string = `${os.homedir()}/.polykey`,
```

Comment:
This is exactly what should be in the `config`. Do not use `env` and `os` stuff around the codebase since it makes it hard to mock. Too many side-effects and hardcoding.